include/simdjson/dom/element.h

```diff
@@ -473,51 +472,51 @@ struct simdjson_result<dom::element> : public internal::simdjson_result_base<dom
   really_inline simdjson_result(dom::element &&value) noexcept; ///< @private
   really_inline simdjson_result(error_code error) noexcept; ///< @private
 
-  inline simdjson_result<dom::element_type> type() const noexcept;
+  really_inline simdjson_result<dom::element_type> type() const noexcept;
```

Comment:
Interesting that we did not have really_inline until now. I wonder whether this had performance impacts.

Reply:
I did this more as a consistency thing in a lot of cases: we are still using just inline in a lot of places we don't need to.